perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -123,26 +200,18 @@ def Run(benchmark_spec):
         If a 4th element is included, it is a dictionary of sample
         metadata.
   """
-  vms = benchmark_spec.vms
-  vm = vms[0]
+  vm = benchmark_spec.vms[0]
   logging.info('FIO running on %s', vm)
-  if FLAGS.against_device:
-    file_path = vm_util.PrependTempDir(FLAGS.fio_jobfile)
-    fio_command = '--filename=%s %s' % (vm.scratch_disks[0].GetDevicePath(),
-                                        FLAGS.fio_jobfile)
-  else:
-    file_path = data.ResourcePath(FLAGS.fio_jobfile)
-    fio_command = '--directory=%s %s' % (
-        vm.GetScratchDir(), FLAGS.fio_jobfile)
-  fio_command = 'sudo %s --output-format=json %s' % (fio.FIO_PATH, fio_command)
+
+  fio_command = 'sudo %s --output-format=json %s' % (fio.FIO_PATH,
+                                                     REMOTE_JOB_FILE_PATH)
```

Comment:
I am concerned about the REMOTE_JOB_FILE_PATH. When you push the path to vm, the user is perfkit. When you ran fio command, you are root(sudo). The home directory might be different, so please use absolute path in REMOTE_JOB_FILE_PATH.